frontend/lib/src/components/elements/ArrowVegaLiteChart/ArrowVegaLiteChart.test.tsx

```diff
@@ -60,30 +63,36 @@ const getProps = (props: Partial<PropsWithHeight> = {}): PropsWithHeight => ({
   ...props,
 })
 
+const getCanvasContext = (): CanvasRenderingContext2D => {
+  const canvas = screen
+    .getByTestId("stArrowVegaLiteChart")
+    // eslint-disable-next-line testing-library/no-node-access
+    .querySelector("canvas") as HTMLCanvasElement
+
+  return canvas.getContext("2d") as CanvasRenderingContext2D
+}
+
 describe("VegaLiteChart Element", () => {
-  it("renders without crashing", () => {
+  it("renders without crashing", async () => {
```

Comment:
don't need async here pretty sure

Reply:
We do. ArrowVegaLite has async activity. See https://github.com/streamlit/streamlit/blob/develop/frontend/lib/src/components/elements/ArrowVegaLiteChart/ArrowVegaLiteChart.tsx#L138